### ISSUE TITLE
These are *fantastic* modules – they are also *tiny*

### ISSUE THREAD

**Issue by vogelscheiss** [nysthi/nysthi#127]
I'm so grateful for all the work that's gone into the NYSTHI modules. They're amazing.

That said, good lord, they're microscopic. Is the functionality is being privileged over the interface design? They're often an absolute bugger to work with, especially in the context of a large patch where zooming way in would make it difficult to interconnect with the rest of the patch. 

I'd say the panels are denser than they need to be. I would love it if all that functionality had more room to breathe. 

(But again, thanks!)

**Comment by nysthi** [comment]
Ciao!

Thank you for your comment. I have many requests to change the size, but that would need time for the designing I don't have. 
I do modules because I like to experiment new ideas, and Rack let me to express those ideas, easily and fast.

Anyway, if you noticed, after I finished the "µ" serie, I've increased everywhere sizes

All the distortion modules are HUGE, SAM is HUGE and so on

I'l try to be "larger" ;)
